synapse/lib/stormlib/stix.py

```diff
@@ -40,6 +48,7 @@ def uuid4(valu=None):
                         ('originates-from', 'location', ':org -> ou:org :hq -> geo:place'),
                         ('targets', 'identity', '-> risk:attack :target:org -> ou:org'),
                         ('targets', 'identity', '-> risk:attack :target:person -> ps:person'),
+                        ('targets', 'vulnerabilty', '-> risk:attack -> it:prod:softver -> risk:hasvuln -> risk:vuln'),
```

Comment:
there are additional pivots through risk:attack which can be added

Reply:
This pivot is too inclusive